code/core/system/proxy-velodyne16/src/velodyne16Decoder.cpp

```diff
@@ -233,7 +263,6 @@ void Velodyne16Decoder::nextString(const string &payload) {
             position += 2;
```

Comment:
In this method, you need to save the time stamp of the current first byte from this turn of the Velodyne sensor. Otherwise, the Velodyne synchronicity is wrong compared to other sensors.